### ISSUE TITLE
Phlex Visable in Plex - Server Install Not Completed - PHP MIBS Issue

### ISSUE THREAD

**Issue by RobotRamone** [d8ahazard/Phlex#150]
Phlex Visable in Plex - Server Install Not Completed

I'm setting up my system.  I'm unable to get the Phlex UI to load properly, Apache Server is running, port forwarding isn't set up correctly yet, Phlex private server doesn't resolve, Phlex public server doesn't resolve.

However, I noticed there's a Phlex logo in Plex running on my TV.

Not sure what's going on.

**Comment by RobotRamone** [comment]
I get this error:

The sockets PHP extension, which is required for Phlex to work correctly, is not loaded. Please enable it in php.ini, restart your webserver, and then reload this page to continue.

So I go to the web address in the PHP.ini file notes and I'm lost.  The recommended file to download is an exe file.  Is that correct?

The MIBS data available in the PHP distribution must be installed.
; See http://www.php.net/manual/en/snmp.installation.php

**Comment by d8ahazard** [comment]
Nope.  Not sure where you'd have PHP installed to, but you need to find
your php.ini file, and then refer to the readme on github for how to enable
php_sockets.

**Comment by RobotRamone** [comment]
Thank you for the response.  I did just that.  I included the only reference to sockets in my above comment.  It says to got to the php.net website.  I'll continue to search for a solution on GitHub, as you recommended.  If you think of a quicker fix the help would be greatly appreciated. 


; The MIBS data available in the PHP distribution must be installed.
; See http://www.php.net/manual/en/snmp.installation.php
;extension=php_snmp.dll

;extension=php_soap.dll
;extension=php_sockets.dll
